src/NHSD.BuyingCatalogue.Ordering.Contracts/NHSD.BuyingCatalogue.Ordering.Contracts.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Can you add this element to the property group?

```xml
<AnalysisMode>AllEnabledByDefault</AnalysisMode>
```

Reply:
Added `AnalysisMode` element